java/main/com/google/privacy/differentialprivacy/BoundedSum.java

```diff
@@ -25,6 +25,8 @@
 import com.google.differentialprivacy.Data.ValueType;
 import com.google.differentialprivacy.SummaryOuterClass.BoundedSumSummary;
 import com.google.protobuf.InvalidProtocolBufferException;
+
+import java.io.Serializable;
```

Comment:
why it was added?

Reply:
My bad, removed. Thanks for spotting it!